include/TimeLineWidget.h

```diff
@@ -180,6 +180,10 @@ public slots:
 	{
 		updatePosition( TimePos() );
 	}
+	void updateSnapSize( const float snapSize )
```

Comment:
Wouldn't `setSnapSize` be a more conventional name?

Reply:
Good point! Fixed.